tests/test_encrypted_fields.py

```diff
@@ -240,3 +243,57 @@ def test_encrypt_public_decrypt_private(self):
             with secret_model() as model:
                 retrieved_secret = model.objects.get(id=secret.id)
                 self.assertEqual(test_val, retrieved_secret.name)
+
+
+class BaseEncryptedFieldTestCase(TestCase):
+
+    @classmethod
+    def setUpClass(cls):  # noqa
+        cls.tmpdir = tempfile.mkdtemp()
+        keyczart.Create(cls.tmpdir, "test", keyinfo.DECRYPT_AND_ENCRYPT,
+                        asymmetric=True)
+        keyczart.AddKey(cls.tmpdir, "PRIMARY", size=4096)
+
+    @classmethod
+    def tearDownClass(cls):  # noqa
+        import shutil
```

Comment:
One last question :-) any reason to do the import here instead of on the top of the file ?

Reply:
there was no reaston - it was just copy&paste ;) you're right so I've moved that import on the top of the test file.

Reply:
👍 
When the commit hits the branch / pr I will merge it.
Thanks as always for your work !

Reply:
you're welcome, the goal is to reach green coverage bandage .. but it starts to be difficult ;)